src/ducks/settings/TogglePane.styl

```diff
@@ -0,0 +1,5 @@
+.TogglePane__main
+    margin-bottom 3rem
+
+.TogglePane__description
+    color coolGrey
```

Comment:
We should have a description label in cozy ui for that.

Reply:
have you a link to the documentation ?